src/app/config/parameters.yml

```diff
@@ -33,7 +33,11 @@ parameters:
 
     zikula.session.name: _zsid
     assets_version: ~ #e.g. 'v1'
-    
+    zikula_asset_manager.combine: true
+    zikula_asset_manager.lifetime: "1 day"
+    zikula_asset_manager.compress: false
+    zikula_asset_manager.minify: false
+
```

Comment:
Why not reusing the existing module vars (which would be more consistent for the user)?